trulens_eval/trulens_eval/schema/feedback.py

```diff
@@ -182,6 +182,9 @@ class FeedbackMode(str, Enum):
     """Evaluate later via the process started by
     `tru.start_deferred_feedback_evaluator`."""
 
+    SERVER = "server"
+    """Evaluate feedback later in server side and not in app environment."""
```

Comment:
Please add a bit more documentation to better distinguish it from DEFERRED.

Reply:
Done